### ISSUE TITLE
Use UI Automation to access Qt applications

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#13930]
### Is your feature request related to a problem? Please describe.
With Qt release 5.11 its MSAA accessibility backend was replaced with UI Automation.
### Describe the solution you'd like
NVDA can now use UI Automation to access Qt applications.
### Describe alternatives you've considered
Both MSAA and UIA can be used, but I don't recommend it. Because the MSAA API is useless in most applications without accessibility work.
### Additional context
Qt applications should be tested by NVDA users after the feature is implemented.

**Comment by CoBC** [comment]
There are still some issues bitween NVDA and Qt. For example if a combobox is editable in a Qt application, we can't see characters with arrow keys with NVDA, working with other screenreaders. Qt is currently fixing some accessibility issues (especially in Qt 6) so if a work can also be done on the NVDA side maybe Qt apps will be more usable in the neer future. I can open a specific issue for the problem describe here with editable comboboxes if necessary.

**Comment by OzancanKaratas** [comment]
@CoBC, What API do the latest versions of TeamTalk use?

**Comment by OzancanKaratas** [comment]
Thanks. I will close this issue. This issue will be reopened in case Qt 6 is not accessible.

@lukaszgo1 and @CoBC, thanks for your replies.
